src/main/java/io/rcktapp/rql/dynamo/DynamoRql.java

```diff
@@ -88,6 +91,65 @@ else if (sk != null && sk.equals(order.col))
       return recursePredicates(predicates, dynamoExpression, andOr, excludeList, 0);
    }
 
+   public DynamoExpression buildDynamoExpressionUsingIndex(Map<String, String> requestParams, DynamoIndex index) throws Exception
```

Comment:
This method is very similar to the buildDynamoExpression.. I'd like to find a way to avoid needing this one and instead expand the main one.